### DISCUSSION TITLE
Unable to get Data from ElasticSearch

### DISCUSSION THREAD

**Discussion by ashokdev7** [Security-Onion-Solutions/securityonion#6894] [Unsupported Versions] [answered]
I am trying to fetch elasticsearch data from a SecurityOnion AWS instance using public ip in my own machine / terminal / CMD.

I am able to ping machine public ip
But can't ping on port 9200 . Telnet shows filtered error . I have enabled port 9200 also on AWS security rules.

Can anyone help me how to do it.

**Comment by RobertClum** [comment]
Filtered response typically points to a proxy or firewall in between, I would double check your firewall rules to make sure its open.

**Comment by dougburks** [comment, marked as answer]
Make sure that you run `so-allow` to allow the traffic through the host based firewall:
https://docs.securityonion.net/en/2.3/so-allow.html
